pages/en/lb4/Using-components.md

```diff
@@ -7,24 +7,42 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Using-components.html
 summary:
 ---
-Components are an important part of LoopBack 4. We are keeping the core small and easy to extend, making it easy for independent developers to contribute additional features to LoopBack. Components are the vehicle for bringing more goodness to your applications.
 
-A typical LoopBack component is an [npm](https://www.npmjs.com/) package exporting a Component class which can be added to your application.
+One of the many ways LoopBack 4 allows for extensibility is through Components.
+A Component makes it easy for independent developers to contribute additional
+features to LoopBack. Components provide a package that allows you to extend
+your Application.
 
-```js
-import {Application} from '@loopback/core';
-import {AuthenticationComponent} from '@loopback/authentication';
+A typical LoopBack component is an [npm](https://www.npmjs.com) package
+exporting a Component class whih can be added to your application.
 
-const app = new Application();
+```ts
+import { RestApplication } from "@loopback/rest";
+import { AuthenticationComponent } from "@loopback/authentication";
+
+const app = new RestApplication();
+// Add component to Application, which provides bindings used to resolve
+// authenticated requests in a Sequence.
 app.component(AuthenticationComponent);
 ```
 
-In general, components can contribute the following items:
+Components can contribute the following items:
```

Comment:
Components can contribute any type of artifacts that can be bound to a context.

Reply:
Current implementation of a Component is limited to the following two out of the box. Additional Artifacts have to enabled via Mixins. We have an issue to add support for artifacts https://github.com/strongloop/loopback-next/issues/1080

I do think Docs should reflect the current state of the framework and hence not listing other Artifacts in this section. Section below covers using Mixins to allow Components to contribute additional Artifacts.